roles/dropwizard-api/files/start-container.j2

```diff
@@ -41,7 +41,9 @@ docker rm -f $api &
 
 # Sometimes docker will still think a container exists after doing "docker rm"
 # Pause the script for a few seconds before creating the container again
-sleep 8
+until [[ ! "$(docker ps -aq -f name=$api)" ]]; do
```

Comment:
I tested this logic on my machine and it worked as expected. A few things to note:
- If the container never gets removed (I've had cases where `docker rm -f` has failed due to some bug in docker), this loop will run forever. Since we are running `docker rm -f &`, its running in the background so we don't know if that command failed due to the container not existing or if there was an error removing the container. I'm thinking we should bundle this `until` loop and the `docker rm -f` command in an `if` block that checks if the container actually exists first before we try to remove it and wait until its removed. That way, if `docker rm -f` failed, we know it's because there was an error removing the container because we already checked that the container exists. If the container doesn't exist in the first place, then great, we can proceed with running it.
- This isn't a big deal, but using the `-f name=foo` filter on `docker ps` will do a substring search of the container name. So if we had `bar-api` and `foo-bar-api`, doing `docker ps -aq -f name=bar-api` would return the container IDs for `bar-api` and `foo-bar-api`. I realize we're doing a similar command for [this line](https://github.com/osu-mist/ansible-roles/blob/59692483ac53d475cf4b039e2c049cab9ae6f31a/roles/dropwizard-api/files/start-container.j2#L70), so this is something we might want to think about improving in the future.